Migrate content and deprecate wiki (#20)
EGI-Foundation/fedcloud-documentation: source/faq.rst

FedCloud Check-in client allows you to get one as needed. Check the
:doc:`auth` guide for more information.

Is OCCI still supported?
^^^^^^^^^^^^^^^^^^^^^^^^

OCCI is now deprecated as API for the EGI Cloud providers using OpenStack. If

> you still rely on OCCI for your access, please contact us for support on the 

Reviewer: maybe would be useful to have a contact email here directly

Author: done

   100IT               org.openstack.nova  https://cloud-egi.100percentit.com:5000/v3/

The providers also generate dynamic information about their characteristics via
the Argo Messaging System which is easily browsable from AppDB.

> 

Reviewer: ```suggestion
```

How can I choose which site to use?
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

Sites offer their resources to users through Virtual Organisations (VO). Before

> you can access a Federated Cloud site you need to join a Virtual Organisation

Reviewer: The more logical flow would be to have users be part of a VO for some reason then getting the VO supported somewhere instead of asking to join a VO to access a specific site.

Author: I rephrased check it if makes sense now

supported by that site.  AppDB shows the supported VOs and for each VO you can
browse the resource providers that support it.

How can I get information about the available VM images?
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

The `Application Database <https://appdb.egi.eu>`_ contains information about
the VM images available in the EGI Cloud. Within the AppDB Cloud Marketplace,
you can look for a VM and get all the information about which VO the VM is
associated, the sites where the VM is available and the endpoints and identifiers
to use it in practice. 

Managing VMs
------------

The disk on my VM is full, how can I get more space?
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

There are several ways to increase the disk space available at the VM. The
fastest and easiest one is to use block storage, creating a new storage disk
device and attaching it to the VM. Check the :doc:`storage` guide for more
information.

> 

Reviewer: ```suggestion
```

How can I keep my data after the VM is stopped?
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> After a VM is stopped and unless backed up in a block storage volume, all data

Reviewer: ```suggestion
After a VM has been stopped and unless backed up in a block storage volume, all data
```

in the VM is destroyed and cannot be recovered. To ensure your data will be
available after the VM is deleted, you need to use some form of persistent
storage.

> 

Reviewer: ```suggestion
```

Reviewer: A link to the related section would help here.
